src/tests/multihost/ad/test_access_control.py

```diff
@@ -12,6 +13,23 @@
 from sssd.testlib.common.exceptions import SSHLoginException
 
 
+def ssh_login(multihost, username):
+    client_hostname = multihost.client[0].sys_hostname
+    client = pexpect_ssh(client_hostname, username, 'Secret123', debug=False)
+    try:
+        client.login()
+    except SSHLoginException:
+        pytest.fail(f'{username} failed to login')
+        return 'false', 'login failed'
+    except pexpect.EOF as err:
+        log_str = multihost.client[0].get_file_contents('/var/log/secure').decode('utf-8')
```

Comment:
The log_str is unused, maybe cat or tail the log file instead so it is shown in the console/test log.

Reply:
Oh, I missed that.